Automattic/wp-calypso
# Add command palette single site commands

Related to
- https://github.com/Automattic/dotcom-forge/issues/5493

## Proposed Changes

* Implemented the commands for the command palette

## Testing Instructions



* Apply the https://github.com/Automattic/jetpack/pull/35635.
* Download the App and upload it to wpcom widgets.
* Make sure to sandbox widgets.
* Go to `/wp-admin` and press CMD + K
* Test all the commands.

![Screenshot 2024-02-27 at 10 02 55](https://github.com/Automattic/wp-calypso/assets/1989914/fddd2ebd-3982-4e85-bf7f-92ec859c9e7a)


## Pre-merge Checklist
